workshops/gloo-mesh-demo/README.md

```diff
@@ -176,14 +188,23 @@ export PATH=$PWD/istio-${ISTIO_VERSION}/bin:$PATH
 istioctl version
 ```
 
-2. Install Istio to each of the remote clusters. If you're using local Kubernetes clusters on a Mac M1 or M2, use [these ARM instructions](problems-istio-arm.md) instead.
+2. Install Istio to each of the remote clusters using the Gloo Mesh Istio Lifecycle Manager and Gateway Lifecycle Manager.
 
 ```sh
-kubectl create namespace istio-gateways --context $CLUSTER1
-istioctl install --set hub=$ISTIO_IMAGE_REPO --set tag=$ISTIO_IMAGE_TAG  -y --context $CLUSTER1 -f install/istio/istiooperator-cluster1.yaml
+kubectl apply -f install/istio/gm-istiod.yaml --context $MGMT
+kubectl apply -f install/istio/gm-ingress-gateway.yaml --context $MGMT
```

Comment:
Where is the east-west gateway installed?

Reply:
Good catch. I had it. Just needed to add it to the README. I'll put it in the inline version.